ios/FluentUI/Core/Theme/Tokens/UIControlTokenResolver.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// Internal extension to `TokenizedControl` that adds information about `configuration`.
+protocol TokenizedUIControlInternal: TokenizedControl {
+    associatedtype ConfigurationType: ControlConfiguration where ConfigurationType.TokenType == TokenType
+
+    /// Contains additional configuration information about the control.
+    var config: ConfigurationType { get }
```

Comment:
Turns out that any control we make that inherits from UIControl isn't really allowed to override state.

Reply:
Okay I'm prepping a PR with a workaround, but let's not block this work on that.